pkg/actuators/machine/actuator.go

```diff
@@ -125,10 +128,24 @@ func (a *Actuator) Update(ctx context.Context, machine *machinev1.Machine) error
 		if err := scope.patchMachine(); err != nil {
 			return err
 		}
-		return a.handleMachineError(machine, err, createEventAction)
+		fmtErr := fmt.Sprintf(reconcilerFailFmt, machine.GetName(), updateEventAction, err)
```

Comment:
isn't this making us lose the error type machineapierros.InvalidMachineConfiguration MachineError?